web/web.ex

```diff
@@ -37,6 +37,7 @@ defmodule App.Web do
       import App.Router.Helpers
       import App.Gettext
       import App.Auth, only: [authenticate_user: 2]
+      require IEx
```

Comment:
I couldn't think of a way to pass IEx down to my controllers only in `test`/`dev` environments